app/celery/letters_pdf_tasks.py

```diff
@@ -1,424 +0,0 @@
-        dvla_filename = 'NOTIFY.{date}.{num:03}.{hash}.ZIP'.format(date=date, num=i + 1, hash=hash)
-
-        current_app.logger.info(
-            'Calling task zip-and-send-letter-pdfs for %d pdfs to upload %s with total size %s bytes',
-            len(filenames),
-            dvla_filename,
-            '{:,}'.format(sum(letter['Size'] for letter in letters)),
-        )
-        notify_celery.send_task(
-            name=TaskNames.ZIP_AND_SEND_LETTER_PDFS,
-            kwargs={'filenames_to_zip': filenames, 'upload_filename': dvla_filename},
-            queue=QueueNames.PROCESS_FTP,
-            compression='zlib',
-        )
-
-
-def group_letters(letter_pdfs):
-    """
-    Group letters in chunks of MAX_LETTER_PDF_ZIP_FILESIZE. Will add files to lists, never going over that size.
-    If a single file is (somehow) larger than MAX_LETTER_PDF_ZIP_FILESIZE that'll be in a list on it's own.
-    If there are no files, will just exit (rather than yielding an empty list).
-    """
-    running_filesize = 0
-    list_of_files = []
-    for letter in letter_pdfs:
-        if letter['Key'].lower().endswith('.pdf') and letter_in_created_state(letter['Key']):
-            if (
-                running_filesize + letter['Size'] > current_app.config['MAX_LETTER_PDF_ZIP_FILESIZE']
-                or len(list_of_files) >= current_app.config['MAX_LETTER_PDF_COUNT_PER_ZIP']
-            ):
-                yield list_of_files
-                running_filesize = 0
-                list_of_files = []
-
-            running_filesize += letter['Size']
-            list_of_files.append(letter)
-
-    if list_of_files:
-        yield list_of_files
-
-
-def letter_in_created_state(filename):
-    # filename looks like '2018-01-13/NOTIFY.ABCDEF1234567890.D.2.C.C.20180113120000.PDF'
-    subfolder = filename.split('/')[0]
-    ref = get_reference_from_filename(filename)
-    notifications = dao_get_notifications_by_references([ref])
-    if notifications:
-        if notifications[0].status == NOTIFICATION_CREATED:
-            return True
-        current_app.logger.info(
-            'Collating letters for %s but notification with reference %s already in %s',
-            subfolder,
-            ref,
-            notifications[0].status,
-        )
-    return False
-
-
-@notify_celery.task(bind=True, name='process-virus-scan-passed', max_retries=15, default_retry_delay=300)
-def process_virus_scan_passed(
-    self,
-    filename,
-):
-    reference = get_reference_from_filename(filename)
-    notification = dao_get_notification_by_reference(reference)
-    current_app.logger.info('notification id %s Virus scan passed: %s', notification.id, filename)
-
-    is_test_key = notification.key_type == KEY_TYPE_TEST
-
-    scan_pdf_object = s3.get_s3_object(current_app.config['LETTERS_SCAN_BUCKET_NAME'], filename)
-    old_pdf = scan_pdf_object.get()['Body'].read()
-
-    try:
-        billable_units = get_page_count(old_pdf)
```

Comment:
`get_page_count` method in the ` app.letters.utils.get_page_count.py` can be removed - it is only called here.